cl/search/models.py

```diff
@@ -545,6 +556,36 @@ class Docket(AbstractDateTimeModel, DocketSources):
         blank=True,
         db_index=True,
     )
+    office_code = models.CharField(
+        help_text="A one digit statistical code (either alphabetic or numeric) "
+        "of the office within the district. In this "
+        "example, 2:07-cv-34911-MJL, the 2 preceding "
+        "the : is the office code.",
+        max_length=1,
```

Comment:
John makes the case that office codes should be more than one character and sure, why not. Want to make them three to provide some headroom?